### PR TITLE
🔧 Add init. and config sqlalchemy

### PR DESCRIPTION
**Pull request by Vaserti**

### CODE REVIEW

File: shortly/app/config/base.py [hunk 1]
```diff
@@ -1,5 +1,46 @@
...
+    # Postgres logs settings
+
+    DB_ECHO_LOGS: bool = True
+
+    class Config:
+        env_file = ".env"
+        case_sensitive = True
+
+
+settings = Settings()
```
Comment (imatw4r) [inline shortly/app/config/base.py]: Please, move instantiation to `app/config/__init__.py`:

```
from .base import Settings

settings = Settings()
```

The reason for that is that if we would ever split Settings to separate classes `SettingsProd` vs `SettingsTest` to for example forbid overwriting of some fields, we could have a logic in `__init__.py` that selects which class gets instantiated as a settings, based on env variable.

File: shortly/app/main.py [hunk 2]
```diff
@@ -1,3 +1,9 @@
 from fastapi import FastAPI
 
+from shortly.app.events import setup_events
+
 app = FastAPI()
+
+# setup FastAPI components
```
Comment (imatw4r) [inline shortly/app/main.py]: I don't think we need this comment here 🤔

File: shortly/app/config/base.py [hunk 3]
```diff
@@ -1,5 +1,46 @@
...
+from typing import Any
+
+from pydantic import BaseSettings, PostgresDsn, SecretStr, validator
 
 
 class Settings(BaseSettings):
-    pass
+    # Postgres connection settings
+
+    POSTGRES_USER: str
```
Comment (imatw4r) [inline shortly/app/config/base.py]: Didn't we agree to nest the settings, so we can use:
```python
settings.db.host
settings.db.port
settings.db.get_async_psql_uri()
```

Comment (Vaserti) [reply shortly/app/config/base.py]: Great idea :). Done

File: shortly/app/main.py (hunk 2 above)
Comment (Vaserti) [reply shortly/app/main.py]: After thinking about it, you're right. 
Function name setup_* itself indicates installations.

Comment (Vaserti) [reply shortly/app/main.py]: Done

File: shortly/app/config/base.py (hunk 1 above)
Comment (Vaserti) [reply shortly/app/config/base.py]: I understand  idea and see the meaning :)

Comment (Vaserti) [reply shortly/app/config/base.py]: Done

File: shortly/app/config/base.py [hunk 4]
```diff
@@ -1,5 +1,50 @@
-from pydantic import BaseSettings
+from pydantic import BaseSettings, PostgresDsn, SecretStr
+
+
+class DatabaseSettings(BaseSettings):
+    # Postgres connection settings
+
+    USER: str
+    PASSWORD: SecretStr
+    DB_NAME: str
```
Comment (imatw4r) [inline shortly/app/config/base.py]: Wouldn't simple "NAME" be enough?
```python
settings.db.name
vs
settings.db.db_name
```

**Review by imatw4r — APPROVED** [review]

File: shortly/app/config/base.py (hunk 4 above)
Comment (Vaserti) [reply shortly/app/config/base.py]: Done

**Pull request merged**